### DISCUSSION TITLE
The form component doesn't work

### DISCUSSION THREAD

**Discussion by carolinamorah** [mdbootstrap/TW-Elements#1681] [Support from community] [answered]
Hello, I'm using the form component for the login on my website, but sometimes the blue border of the input stops working, and the label overlaps the input and covers the content. How do I solve this issue? How can I make the label animation and the blue border when focusing on the input continue to work? Sometimes, when I reload the page, it stops working.

**Comment by juujisai** [comment, marked as answer]
Hi, could you give us more info like:

- When does this happen,
- Are you using the `ES` format? If yes then are you using the `initTE` method?
- Is there some specific window or input width where you are experiencing this issue or is it everytime,
- Maybe you could show us some code, or create a snippet? https://tailwind-elements.com/snippets/


Best Regards!
